pkg/ytconfig/cri.go

```diff
@@ -18,6 +18,7 @@ type CRIConfigGenerator struct {
 	Isolated       bool
 	StoragePath    *string
 	MonitoringPort int32
+	HasGPU         bool
```

Comment:
I would suggest calling it smth like `autoConfigureGPU` to explicitly indicate that there will be an attempt to do something automatically (possibly unsuccessful, if we have not nvidia device)